src/components/pages/files/views/Files.tsx

```diff
@@ -10,17 +22,21 @@ const DashboardFile = dynamic(() => import('@/components/file/DashboardFile'), {
   loading: () => <Skeleton height={350} animate />,
 });
 
+const PER_PAGE_OPTIONS = [15, 30, 45];
```

Comment:
```suggestion
const PER_PAGE_OPTIONS = [9, 12, 15, 30, 45];
```

just some smaller options as well